### PR TITLE
Also mount API in /api

### PR DESCRIPTION
**Pull request by sathieu**
See https://github.com/InseeFrLab/helm-charts/issues/23

### CODE REVIEW

**Comment by sathieu** [comment]
@olevitt @fcomte Please review 🙏 !

J'en ai discuté déjà avec @fcomte.

**Comment by garronej** [comment]
LGTM

File: onyxia-api/src/main/kotlin/fr/insee/onyxia/api/controller/pub/HealthcheckController.kt [hunk 1]
```diff
@@ -7,7 +7,7 @@ import org.springframework.web.bind.annotation.RestController
 
 @RestController
 @Tag(name = "Public")
-@RequestMapping("/public")
+@RequestMapping({"/api/public", "/public"})
```
Comment (sathieu) [inline onyxia-api/src/main/kotlin/fr/insee/onyxia/api/controller/pub/HealthcheckController.kt:10]: @garronej I'm sorry, I've broken the build: https://github.com/InseeFrLab/onyxia-api/runs/4365099636?check_suite_focus=true#step:5:1574

I don't know anything about kotlin ...

Comment (garronej) [reply onyxia-api/src/main/kotlin/fr/insee/onyxia/api/controller/pub/HealthcheckController.kt:10]: Me neither lol, I'm looking into it

Comment (garronej) [reply onyxia-api/src/main/kotlin/fr/insee/onyxia/api/controller/pub/HealthcheckController.kt:10]: image

Comment (garronej) [reply onyxia-api/src/main/kotlin/fr/insee/onyxia/api/controller/pub/HealthcheckController.kt:10]: it's [here](https://github.com/InseeFrLab/onyxia-api/blob/714c65ca74180980b7093f123e9f094dd5f40587/onyxia-api/src/main/kotlin/fr/insee/onyxia/api/controller/pub/HealthcheckController.kt#L10)

**Pull request merged**